lms/templates/main.html

```diff
@@ -175,6 +175,8 @@
   % endif
 
   <%block name="js_extra"/>
+
+  <script type="text/javascript" src="${settings.STATIC_URL}js/utils/dropdown.js"></script>
```

Comment:
Just confirming that @AlasdairSwan is okay with including this separate file on all LMS templates. Cool?